src/main/java/com/demo/controller/AuthRestAPIs.java

```diff
@@ -133,10 +134,12 @@
   }
 
 
-  @GetMapping(value = "/forgot-password")
-  public ResponseEntity<?> forgotUserPassword(@RequestParam("gmail") String gmail) {
+  @PostMapping(value = "/forgot-password")
+  public ResponseEntity<?> forgotUserPassword(@RequestBody String gmail) {
     User existingUser = userRepository.findByEmailIgnoreCase(gmail);
```

Comment:
Controller gọi tầng repository. Yêu cầu xem lại toàn bộ code, tầng controller gọi service, service gọi repository!!!

@duongtiendat1990 @khuongduy1484